### PR TITLE
Add project euler

### PR DESCRIPTION
**Pull request by darakian**
This PR adds a project euler section to the repo and includes an example solution for problem 1
This mirrors the python repo seen here
https://github.com/TheAlgorithms/Python/tree/master/project_euler

### CODE REVIEW

File: src/project_euler/problem_01/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let mut the_sum = 0;
+    for i in 1..1000{
+        if i%3==0 || i%5 == 0{
```
Comment (bh2smith) [inline src/project_euler/problem_01/src/main.rs:4]:
```suggestion
        if i % 3 == 0 || i % 5 == 0{
```

**Comment by bh2smith** [comment]
May also want to include a new section in the README for Project Euler.

**Comment by darakian** [comment]
@bh2smith I've updated the PR with some readme content as well as different spacing.

**Review by bh2smith — DISMISSED** [review]
Looks good to me although I am not a maintainer of this project. Will probably have to be approved by someone with permission to merge!

**Comment by AnshulMalik** [comment]
Hi @darakian ,

Please create the directory as `_1` only. See this PR for example: https://github.com/TheAlgorithms/Rust/pull/145

**Comment by darakian** [comment]
> Hi @darakian ,
> 
> Please create the directory as `_1` only. See this PR for example: #145

Updated with `_1` as the dir name

**Comment by MonliH** [comment]
It would be nice if this was just its own module, not a crate, as mentioned in #145.

**Comment by darakian** [comment]
Sure, whatever. Updated.

**Comment by MonliH** [comment]
Awesome, thanks!

**Pull request closed without merging**